core/src/main/scala/org/apache/spark/deploy/history/EventLogFileCompactor.scala

```diff
@@ -158,6 +159,8 @@ class EventLogFileCompactor(
       )
     }
     logWriter.stop()
+    val duration = System.currentTimeMillis() - startTime
```

Comment:
How about directly using `Utils.timeTakenMs`?

Reply:
I see, using `Utils.timeTakenMs` is also a good choice.